shells/firefox/build.js

```diff
@@ -25,13 +25,13 @@ const main = async () => {
   console.log(chalk.gray('\n# Go to the unpacked directory:'));
   console.log(`cd ${unpackedPath}`);
   console.log(chalk.gray('\n# And launch Firefox with the extension enabled:'));
-  console.log('web-ext run');
+  console.log('yarn run start:firefox');
```

Comment:
Maybe it's okay for us to remove the manual `web-ext` instructions entirely in favor of just calling out the NPM script. What do you think? 😄